Application/Client/src/Chef.cpp

```diff
@@ -0,0 +1,178 @@
+#include <iostream>
+#include <iomanip>
+#include <limits>
+#include <stdexcept>
+#include "Chef.h"
+#include "Utils.h"
+#include "ClientSocket.h"
+#include "RequestType.h"
+
+enum class ChefOptions {
+    DISPLAY_MENU,
+    VIEW_SPECIFIC_DATE_MENU,
+    ROLLOUT_NEXT_DAY_MENU,
+    VIEW_NEXT_DAY_VOTES,
+    VIEW_FEEDBACK_RATINGS,
+    LOGOUT
+};
+
+Chef::Chef(std::string username) : logged_username(username) {
+}
+
+void Chef::chefLandingPage() {

```

Comment:
Check similarly here and other places for proper naming. This should also be a verb.